src/app/common/hooks/use-key-actions.ts

```diff
@@ -55,6 +57,8 @@ export function useKeyActions() {
 
       async signOut() {
         await clearWalletSession();
+        networksActions.changeNetwork('mainnet');
+        void analytics.track('change_network', { id: 'mainnet' });
```

Comment:
I don't think we want an analytics event here. The user isn't actually changing network, and this makes it seem as so.